functions/src/offers/index.ts

```diff
@@ -24,21 +23,17 @@ const queueStatusUpdateTriggers = async (
     offerBefore?.status !== OfferStatus.accepted &&
     offerAfter?.status === OfferStatus.accepted
   ) {
+    // Update Request with new status and CAV.
     operations.push(offerAfter.requestRef.update({
       cavUserRef: offerAfter.cavUserRef,
-      status: RequestStatus.ongoing
+      status: RequestStatus.ongoing,
     }));
-    let offersToReject = await db.collection('offers').where('requestRef', '==', offerAfter.requestRef).get();
-    offersToReject.forEach(offer => {
-      if(offer.ref !== change.after.ref){
-        operations.push((async () => {
-          await offer.ref.update({
-            status: OfferStatus.rejected
-          });
-          Promise.resolve();
-        })()); 
-      }
-    });
```

Comment:
We didn't need to do this as it would be quite expensive and we don't need to `reject` those offers, particularly if the first one falls through.

Reply:
okay. I looked at the status Object and saw that there was a `rejected` status.
So I assumed that the requirement was to invalidate all offers and only keep one as accepted.
This makes sense